packages/client-core/src/user/components/UserMenu/menus/AvatarCreatorMenu2.tsx

```diff
@@ -178,24 +186,14 @@ const AvatarCreatorMenu = (selectedSdk: string) => () => {
     }
     loading.set(LoadingState.Uploading)
 
-    const canvas = document.createElement('canvas')
```

Comment:
we still need to use this for non-RPM avatars, please keep the original functionality for user-uploaded avatars (which is under a feature flag)